_data/kit_versions.yml

```diff
@@ -1,11 +1,2 @@
 # The latest version should be the first in this file.
-- version: 2022.2.0
-  released: 2022-04-18
-- version: 2022.1.0
-  released: 2022-02-04
-- version: 2022.0.2
-  released: 2021-11-23
-- version: 2022.0.1
-  released: 2021-11-20
-- version: 2022.0.0
-  released: 2021-11-13
+[]
```

Comment:
Could we have a comment in here with the structure of a version (version + released)? Just thinking will save some time and uncertainty when adding first update